Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM node:5.11.1
 
-ADD ./package.json /code/package.json
 WORKDIR /code
+
+COPY ./package.json /code/package.json
```

Comment:
`npm install` wasn't being cached for me when using `ADD`. [Docker now encourages `COPY` instead of `ADD`](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/add-or-copy)